src/ui/ResponsiveComponents/ResponsiveComponentsManager.ts

```diff
@@ -65,7 +65,6 @@ export class ResponsiveComponentsManager {
         let responsiveComponentsManager = _.find(this.componentManagers, componentManager => root.el == componentManager.coveoRoot.el);
         if (!responsiveComponentsManager) {
           responsiveComponentsManager = new ResponsiveComponentsManager(root);
-          this.componentManagers.push(responsiveComponentsManager);
```

Comment:
Hmm, I'm not sure about this change. Could there be a case where someone wants to instantiate a `ResponsiveComponentsManager` without modifying the static property? What was the reason for needing this now?

Reply:
As a developer, when I call `ResponsiveComponentManager.resizeAllComponentsManager`, I'm expecting it does what it says, which is currently not the case.

The initial motivation for that was, at first, that I cannot access `this.componentManagers` which blocked me for the UT of `resizeAllComponentsManager`.

Then, I checked the usage of the class, it's constructor it's not used anywhere else than the UT and the `register` method, because all Responsive Component used the register method.

A case where one could want to instantiate a ResponsiveComponentManager would be if one would like to instantiate it and do not be 'resized' when the ResponsiveComponents are done initializing, however, I think that'd qualify as a misuse of the class. (I think that my first paragraph, i.e., that the method does what its name explicitly says is more important)

Reply:
Ok, I see thanks. Could we include a test to assert the new behaviour where on instantiating the class, we push the instance into the static property? It would help avoid an unintended breaking change in the future.

Reply:
It's already tested as a side-effect by the UT I'm adding:
If we're not adding the instance into the static property, calling `resizeAllComponentsManager` will not call the `handleResizeEvent` spy which come from directly from a constructor, and that would fail `expect(handleResizeEvent).toHaveBeenCalled();`